src/snowflake/cli/api/console/console.py

```diff
@@ -70,28 +66,21 @@ def _format_message(self, message: str, output: Output) -> Text:
     @contextmanager
     def phase(self, enter_message: str, exit_message: Optional[str] = None):
```

Comment:
Looks like we never checked that, but shouldn't we disallow phase when within a step? If so, could you please fix the implementation?

Reply:
right, steps don't have a duration. Ignore me :)